examples/pom.xml

```diff
@@ -78,7 +78,7 @@
             <plugin>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <additionalJOption>--no-module-directories</additionalJOption>
+                    <additionalJOption>${maven-javadoc-plugin.option}</additionalJOption>
```

Comment:
I've refactored this a bit to make it possible to build the CDM with Java 17 (as well as Java 11). The `--no-module-directories` option was removed in Java 13 (it is assumed implicitly from then onwards), so I've factored this option out in a Maven property that is only set when Java is 12 or below (using a profile).